docs/ocis/development/beta-testplan.md

```diff
@@ -0,0 +1,293 @@
+| Move a folder inside another.                                                                                      | No problems while moving the folder.                                   |         |
+| open images in mediaviewer                                                                                         | files are displayed  correctly.                                        |         |
+| open videos in mediaviewer                                                                                         | files are displayed  correctly.                                        |         |
+| switch through videos and images in mediaviewer                                                                    | files are displayed  correctly.                                        |         |
+| Share a file by public link.                                                                                       | Link is created and can be accessed.                                   |         |
+| Share a folder by public link.                                                                                     | Link is created and can be accessed.                                   |         |
+| Share a file with another user.                                                                                    | It is shared correctly.                                                |         |
+| Share a folder with another user.                                                                                  | It is shared correctly.                                                |         |
+| Share a file with a group.                                                                                         | It is shared correctly.                                                |         |
+| Share a folder with a group.                                                                                       | It is shared correctly.                                                |         |
+| Share a folder with userB giving edit permissions. As userB do CRUD operations on items inside the received folder | userB doesn't find any problem while interacting with files.           |         |
+| Use your mobile device to access the UI                                                                            | All elements reachable                                                 |         |
+| do tests mentioned in the [spaces](#spaces) section using the web-UI                                               |                                                                        |         |
+
+## Desktop Client
+
+Prerequisite:
+- [connect ocis to your preferred LDAP server](https://owncloud.dev/ocis/deployment/ocis_ldap/)
+- create users and groups in LDAP
+- use your preferred OS for the desktop client
```

Comment:
maybe also give to users link-instruction how configure the desktop client.

Reply:
some details added